parsers/s01-parse/LePresidente/gitea-logs.yaml

```diff
@@ -0,0 +1,26 @@
+onsuccess: next_stage
+#debug: false
+name: LePresidente/gitea-logs
+filter: "evt.Parsed.program == 'gitea'"
+description: "Parse gitea logs"
+pattern_syntax:
+  CUSTOMDATE: "%{DATE_X} %{TIME}"
+nodes:
+  - grok:
+      pattern: '^%{CUSTOMDATE:timestamp}.*?Failed authentication attempt for (%{EMAILADDRESS:email}|%{USERNAME:username}) from %{IP:remote_ip}:%{NUMBER:remote_port}: user does not exist'
+      apply_on: message
+      statics:
+        - meta: log_type
+          value: gitea_failed_auth
+
+statics:
+    - meta: service
+      value: gitea
+    - meta: user
+      expression: "evt.Parsed.username"
+    - meta: user
```

Comment:
the `evt.Meta.User` is overwritten by the email address

Reply:
I think this is right, evt.Meta.User will either be the username or email address in the log line, the scenenario then can be same for both instances

Reply:
The Grok node uses an if statement so only one will be returned as well (%{EMAILADDRESS:email}|%{USERNAME:username})
if email is empty it will fill username instead

The test passes with this setup currently and the parser.assert shows correctly.

Reply:
I could change it to greedydata if you would prefer?

Reply:
Hello @LePresidente , i missread the grok pattern, i think its ok :)